jobrunner/run.py

```diff
@@ -82,6 +82,10 @@ def handle_jobs(api: Optional[ExecutorAPI]):
                 # workspace. This gives a fairer allocation of capacity among
                 # workspaces.
                 running_for_workspace[job.workspace],
+                # prioritise db jobs over cpu jobs, as they a) have their own
+                # limit and b) the system usually benefits from getting these
+                # done before cpu jobs
+                0 if job.requires_db else 1,
```

Comment:
this change seems like it's been pulled in accidentally from another commit?

Reply:
Ah, yes, will drop thanks